apps/ledger-live-desktop/src/renderer/bridge/proxy.js

```diff
@@ -91,18 +91,16 @@ export const getAccountBridge = (
       }),
     );
   };
-
+  const bridge = bridgeImpl.getAccountBridge(account, parentAccount);
   const receive = (account, arg) =>
     command("AccountReceive")({
       account: toAccountRaw(account),
       arg,
     });
 
-  const createTransaction = a =>
-    bridgeImpl.getAccountBridge(account, parentAccount).createTransaction(a);
+  const createTransaction = a => bridge.createTransaction(a);
```

Comment:
a is for a account ? can you add some jsdoc or more explicit name ?

Reply:
I feel like it's ok since it's in a predicate, but i edited it i don't mind